src/components/Blasts/Blasts.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald:300,400');
```

Comment:
**Don't separate component and styles**

 + Each component style *MUST* be define within that component in JS and loaded using `withStyles(styles)(Component)`.
 + Site wide resources like fonts should be loaded from one location only: `main.css` in your case.

Reply:
On it